payment/controllers.go

```diff
@@ -581,3 +590,42 @@ func MerchantTransactions(service *Service) handlers.AppHandler {
 		return nil
 	})
 }
+
+// VerifyCredentialsRequest includes an order id and opaque subscription credential blob
+type VerifyCredentialsRequest struct {
+	OrderID     uuid.UUID                  `json:"orderId" valid:"-"`
+	Credentials []cbr.CredentialRedemption `json:"credentials" valid:"-"`
```

Comment:
I was thinking it might be useful to make the credential being redeemed a single opaque b64 encoded string. This would make it so that we could do schema evolution of the credential(s) without having to change e.g. the subscription server. wdyt?

Reply:
I can implement that for sure,  

Probably don't need to bulk redeem here yet either, I'll use the singular `RedeemCredential`, I imagine Brave together will be single redemptions to begin

Reply:
Sounds good - I think we could even just go with even more generic `Credential` because in the case of a future subscription credential it may not actually be redeeming anything.

Also we have flexibility of the schema within the base64 blob - so could likely in the future allow for having a single "credential" actually involve multiple tokens.